client/src/ui/PageFooter.tsx

```diff
@@ -90,77 +107,35 @@ export const PageFooter = () => {
       <div
         style={{
           display: "flex",
-          alignItems: "center",
+          justifyContent: "flex-end",
+          gap: "100px", // Space between icons
         }}
       >
-        <Link href="#">
-          <Image
-            src="/x-logo/X-logo-black.png"
-            alt="X"
-            width={24}
-            height={24}
-            style={{
-              margin: "0 40px",
-              opacity: 0.7,
-              transition: "opacity 0.3s ease",
-            }}
-            onMouseEnter={(e) => (e.currentTarget.style.opacity = "1")}
-            onMouseLeave={(e) => (e.currentTarget.style.opacity = "0.7")}
-          />
-        </Link>
-        <Link href="#">
-          <Image
-            src="/Instagram-logo/Instagram_Glyph_Black.svg"
-            alt="Instagram"
-            width={24}
-            height={24}
-            style={{
-              margin: "0 40px",
-              opacity: 0.7,
-              transition: "opacity 0.3s ease",
-            }}
-            onMouseEnter={(e) => (e.currentTarget.style.opacity = "1")}
-            onMouseLeave={(e) => (e.currentTarget.style.opacity = "0.7")}
-          />
-        </Link>
-        <Link href="#">
-          <Image
-            src="/LinkedIn-logo/In-Black.png"
-            alt="LinkedIn"
-            width={24}
-            height={24}
-            style={{
-              margin: "0 40px",
-              opacity: 0.7,
-              transition: "opacity 0.3s ease",
-            }}
-            onMouseEnter={(e) => (e.currentTarget.style.opacity = "1")}
-            onMouseLeave={(e) => (e.currentTarget.style.opacity = "0.7")}
-          />
-        </Link>
-        <Link href="#">
-          <Image
-            src="/Meta-logo/Meta-logo.png"
-            alt="Meta"
-            width={24}
-            height={24}
-            style={{
-              margin: "0 40px",
-              opacity: 0.7,
-              transition: "opacity 0.3s ease",
-            }}
-            onMouseEnter={(e) => (e.currentTarget.style.opacity = "1")}
-            onMouseLeave={(e) => (e.currentTarget.style.opacity = "0.7")}
-          />
-        </Link>
+        {images.map((image, index) => (
```

Comment:
~~You can pass the name of the image in the key prop for the Link element since it's unique within the array.~~ You can use index as the value to pass down to prop. However just keep in mind that using index from the array as the value to pass into the key props is not best practice. Especially later down the line when we implement logic to manipulate the array, React might not the render the content properly. 

I'll leave the documentation on why not to use indexes as keys in this situation. However this works fine
https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key